ex04.ex

```diff
@@ -75,8 +77,32 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
+def even_odd(list) do
```

Comment:
Professor, I really has lost marks in many, would you be able to relax losing marks for this assignment regarding this.  I am from different major Datacenter Systems Engineering, I never had coded and understand the requirement now.